setup.py

```diff
@@ -1,191 +1,242 @@
-# Copyright (c) 2008-2013 by Enthought, Inc.
+# Copyright (c) 2008-2019 by Enthought, Inc.
 # All rights reserved.
+
+from __future__ import absolute_import, print_function
```

Comment:
Actually, let's just ditch both of these. Neither has any significant effect in this context, and we'll be dropping Python 2 support soon enough.